mmv1/third_party/terraform/website/docs/d/apphub_application.html.markdown

```diff
@@ -0,0 +1,164 @@
+---
+subcategory: "App Hub"
+description: |-
+  Application is a functional grouping of Services and Workloads that helps achieve a desired end-to-end business functionality.
+---
+
+# google\_apphub\_application
+
+Application is a functional grouping of Services and Workloads that helps achieve a desired end-to-end business functionality. Services and Workloads are owned by the Application.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_application" "application" {
+  project = "project-id"
+  application_id = "application"
+  location = "location"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - The host project of the application.
+* `application_id` - (Required) The user-defined identifier of the application.
+* `location` - (Required) The location of the application.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
```

Comment:
Instead of listing out each attribute from the original resource, can we just refer to the resource's documentation like this: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/d/sourcerepo_repository.html.markdown?plain=1#L31

Reply:
this will require fewer updates in the future and protect against drift.

Reply:
Done